src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -392,5 +398,10 @@ trait WorkflowComponent {
         findWorkflowMessagesById(id).delete andThen
           findInputResolutionsByWorkflowId(id).delete
     }
+
+    def deleteFailureResolutions(workflowFailureId: Long): DBIOAction[Int, NoStream, Read with Write] = {
+      deleteWorkflowFailureAttributes(workflowFailureId) andThen
```

Comment:
Q: `andThen` will only execute if the previous action completed; is this the behaviour you want during a failure case? Might you want `cleanUp`? (Asking because I don't know, not because I think you're wrong.)